### PR TITLE
Fix workflows

### PR DESCRIPTION
**Pull request by daroga0002**
### Type of change:

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches
- [ ] Documentation
- [ ] Refactor
- [x] Chore
- [x] CI/CD or Tests

### What this PR does / why we need it:

This is fiing some issues related to bad workflows (syntax error in `type` instead `types`. It also bumps actions versions to newest available. Additionally update ginko dependency which is causing CI to fail

### Pre-submission checklist:

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

**Comment by daroga0002** [comment]
Waiting for workflows approval and then I will see does this works correclty or not (if not then I will fix it)

**Review by tao12345666333 — APPROVED** [review]
LGTM

Thanks!!

**Review by lingsamuel — APPROVED** [review]

**Review by AlinsRan — APPROVED** [review]

**Pull request merged**
